R/flex_dashboard.R

```diff
@@ -114,6 +118,7 @@ flex_dashboard <- function(fig_width = 6.0,
                            md_extensions = NULL,
                            pandoc_args = NULL,
                            devel = FALSE,
+                           auto_reload = TRUE,
```

Comment:
maybe name this `resize_reload` instead of `auto_reload`?

Reply:
Sounds good. Makes the name of the argument more self-explanatory as to it's function.